tardis/io/model_reader.py

```diff
@@ -235,6 +244,20 @@ def read_artis_density(fname):
     return time_of_model, velocity, mean_density
 
 
+def read_cmfgen_density(fname):
+
+    df = pd.read_csv(fname, comment='#', delimiter='\s+', skiprows=1)
+    velocity = (df['Velocity'].values[::-1] * u.km / u.s).to('cm/s')
```

Comment:
@wkerzendorf  For CMFGEN files , I think the convention for `velocity` is different . For ex-

- In CMFGEN files velocity values are in `decreasing` order. [CMFGEN file](https://github.com/vg3095/decay/blob/master/DDC15_SN_HYDRO_DATA_0.976d) 
`6.9999926E+04   6.9999184E+04 ........   8.7744269E+02   8.7166905E+02`

- In TARDIS files, velocity values  are in `ascending` order. [TARDIS file](https://github.com/tardis-sn/tardis/blob/master/tardis/io/tests/data/density.dat) 
`9000.0000 10500.000 ......................... 21000.000 22500.000`

- So, when I did not reverse velocity array here(for CMFGEN case), it was not parsing due to this part of code( because I think the convention for velocity is different for CMFGEN case) 
```
    v_inner = velocity[:-1]
    v_outer = velocity[1:]

    invalid_volume_mask = (v_outer - v_inner) <= 0
```

- So, I have reversed it here, so that CMFGEN velocity values can be in ascending order as well 

I don't know , if I am right, so can you check this ?